lib/proxy.rb

```diff
@@ -60,6 +60,25 @@ def process_token_or_authenticate!(env)
     authenticate!(env) unless content_id
   end
 
+  def set_auth_bypass_cookie(response, env)
+    request = Rack::Request.new(env)
+    return response unless request.params['token']
+
+    # Override any existing token, we don't really care at this point if the
+    # token is valid that's up to the consuming app to validate
+    Rack::Utils.set_cookie_header!(
+      response[1],
+      'auth_bypass_token',
+      {
+        value: request.params['token'],
+        path: '/',
+        domain: '.' + Plek.new.external_domain,
```

Comment:
Nice one :-). I'm curious why we need the dot before the domain to include subdomains - the mozilla docs say the subdomains are always included?

Reply:
Yeah I wasn't too sure about whether to add it or not. You're right that currently the indication is that the behaviour without the prefix is meant to be the same - but what I don't know if that's fully consistent so felt this older way was safer.